### PR TITLE
update UIPickerViewDelegate

### PR DESCRIPTION
**Pull request by nicolaouG**
To show only selected countries (with setPickList)

### CODE REVIEW

**Comment by Sharkesm** [comment]
@nicolaouG What's the reason behind updating this part of the codebase?

**Comment by nicolaouG** [comment]
When, for example, I initialize a picker and `setPickList(codes: "CY", "SK", "GB", "GR")`, the pickList gets updated correctly with those 4 desired countries but the picker shows the first 4 countries in the whole countries-list which are not the ones I specify. This is due to this line in the pickerDelegate's viewForRow: `let country = CountryManager.shared.countries[row]`.
If there is another way that i missed, let me know... I just fixed this in my project and so I share my solution with you in case you didn't know about it.

**Comment by Sharkesm** [comment]
@nicolaouG Thanks for the explanation will review this tomorrow :)

**Comment by SURYAKANTSHARMA** [comment]
> @nicolaouG Thanks for the explanation will review this tomorrow :)

@Sharkesm any update?

File: CountryPicker/CountryPicker/PickerView/CountryPickerView.swift [hunk 1]
```diff
@@ -129,7 +129,8 @@ extension CountryPickerView: UIPickerViewDelegate {
             reuseableView = ComponentView(frame: rect)
         }
         
-        let country = CountryManager.shared.countries[row]
+//         let country = CountryManager.shared.countries[row]
+        let country = pickList[row]
```
Comment (SURYAKANTSHARMA) [inline CountryPicker/CountryPicker/PickerView/CountryPickerView.swift:133]: why you had left commented code? 
Any reason for the change, please

File: CountryPicker/CountryPicker/PickerView/CountryPickerView.swift [hunk 2]
```diff
@@ -142,7 +143,8 @@ extension CountryPickerView: UIPickerViewDelegate {
     }
     
     public func pickerView(_ pickerView: UIPickerView, didSelectRow row: Int, inComponent component: Int) {
-        let country = CountryManager.shared.countries[row]
+//         let country = CountryManager.shared.countries[row]
+        let country = pickList[row]
         didSelectCountryCallback?(country)
```
Comment (SURYAKANTSHARMA) [inline CountryPicker/CountryPicker/PickerView/CountryPickerView.swift:148]: same

**Review by SURYAKANTSHARMA — CHANGES_REQUESTED** [review]

File: CountryPicker/CountryPicker/PickerView/CountryPickerView.swift (hunk 1 above)
Comment (nicolaouG) [reply CountryPicker/CountryPicker/PickerView/CountryPickerView.swift:133]: No reason at all. Do you want me to remove the commented lines?

Comment (nicolaouG) [reply CountryPicker/CountryPicker/PickerView/CountryPickerView.swift]: (Done)

**Review by SURYAKANTSHARMA — APPROVED** [review]
👍 Thanks

**Comment by SURYAKANTSHARMA** [comment]
> When, for example, I initialize a picker and `setPickList(codes: "CY", "SK", "GB", "GR")`, the pickList gets updated correctly with those 4 desired countries but the picker shows the first 4 countries in the whole countries-list which are not the ones I specify. This is due to this line in the pickerDelegate's viewForRow: `let country = CountryManager.shared.countries[row]`.
> If there is another way that i missed, let me know... I just fixed this in my project and so I share my solution with you in case you didn't know about it.

Got it thanks 💯 🥇 . Checked as well cc @hardeep-singh

**Comment by Sharkesm** [comment]
> > @nicolaouG Thanks for the explanation will review this tomorrow :)
> 
> @Sharkesm any update?

LGTM 🌟

**Comment by farasyidk** [comment]
Sorry, on cocoapod has not added the above changes @SURYAKANTSHARMA 
thanks

**Pull request merged**